servicecontrol/troubleshooting.md

```diff
@@ -113,3 +113,33 @@ The installer is [code signed](https://en.wikipedia.org/wiki/Code_signing), but
 Although the installer is code signed correctly with a certificate owned by "NServiceBus Ltd", SmartScreen will block it from running until Microsoft has built enough "trust" in the certificate. One of the main inputs to building that trust is when users grant permission to run the installer. To grant permission to run the installer, click "Run Anyway". This will no longer be required when Microsoft decides to trust the certificate.
 
 When building ServiceControl, all build artifacts are virus scanned to ensure no viruses or malware are shipped with the installer packages.
+
+## Indexes get corrupted
+
+Sometimes the following error can be observed:
+
+    Raven.Abstractions.Exceptions.IndexDisabledException: The index has been disabled due to errors
+
```

Comment:
```suggestion
```txt
    Raven.Abstractions.Exceptions.IndexDisabledException: The index has been disabled due to errors
```
```